StitchCore/StitchCoreTests/AuthTests.swift

```diff
@@ -115,4 +115,42 @@ class AuthTests: XCTestCase {
 
         wait(for: [exp], timeout: 10)
     }
+    
+    func testMultipleLoginSemantics() throws {
```

Comment:
I suppose there isn't really a good way to test logout failing with our current setup unless a mocking library exists.

Reply:
Correct ... I may just push the Admin client forward sooner than later at this point. These tests are only going to be flimsy until then.

This change will come alongside a mocking framework (which we'll have to pick) and a matcher framework (https://github.com/Quick/Nimble) which will make our testing much more robust.